types/node/http.d.ts

```diff
@@ -1,8 +1,8 @@
 declare module "http" {
     import * as events from "events";
-    import * as net from "net";
     import * as stream from "stream";
     import { URL } from "url";
+    import { Socket, Server as NetServer } from "net";
```

Comment:
In my opinion this is not improving readability. But feel free to keep it.

Reply:
Might revamp this at some point in a bigger PR, personally it avoids trying to navigate you to the aggregate import when using `go to definition`, feel absolutely free to streamline this at some point.